src/src/SortedArrayView.hpp

```diff
@@ -74,8 +75,11 @@ class SortedArrayView
    * @brief Default move constructor, performs a shallow copy.
    * @param [in/out] src the SortedArray to be moved from.
    */
-  inline
-  SortedArrayView( SortedArrayView && src ) = default;
+  LVARRAY_HOST_DEVICE inline
+  SortedArrayView( SortedArrayView && src )
```

Comment:
isn't this the default?

Reply:
Not quite, the default wouldn't set `src.m_size` to zero.

Reply:
duh